### PR TITLE
feat: remove contract activation control by actor

### PR DESCRIPTION
**Pull request by brew0722**
## Description

closes: #467 

## Motivation and context
- remove contract access control by actor
- remove contract status access config

## How has this been tested?
- re-add inactive contract tests using gov proposal

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [x] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ## [Unreleased](https://github.com/line/lbm-sdk/compare/v0.45.0-rc0...HEAD)
 
 ### Features
+* (x/wasm) [\#470](https://github.com/line/lbm-sdk/pull/470) remove contract activation control by actor
```
Comment (brew0722) [inline CHANGELOG.md:41]: Modifications due to policy changes. 
So, it was judged that it was not a bugfix, and it was judged as a feature.

File: x/wasm/keeper/keeper_test.go [hunk 2]
```diff
@@ -1489,9 +1380,9 @@ func TestClearContractAdmin(t *testing.T) {
 	}
 }
 
-func TestClearContractAdminInactiveContract(t *testing.T) {
+func TestExecuteInactiveContract(t *testing.T) {
```
Comment (loloicci) [inline x/wasm/keeper/keeper_test.go:1383]: Change the name to what tells the test expects the execution fails. (This test tries to change the contract status via non-gov and fails, right?)

File: x/wasm/keeper/keeper_test.go [hunk 3]
```diff
@@ -1501,33 +1392,74 @@ func TestClearContractAdminInactiveContract(t *testing.T) {
...
-	params.ContractStatusAccess = types.AccessTypeOnlyAddress.With(creator)
-	wasmKeeper.setParams(ctx, params)
+	// execute inactive contract through manual
 	err = keeper.UpdateContractStatus(ctx, addr, creator, types.ContractStatusInactive)
+
+	// Contract status can only be changed through a gov proposal.
+	require.True(t, sdkerrors.ErrUnauthorized.Is(err), "expected %v but got %v", sdkerrors.ErrUnauthorized, err)
+}
+
+func TestMigrateInactiveContract(t *testing.T) {
```
Comment (loloicci) [inline x/wasm/keeper/keeper_test.go:1417]: ditto.

File: x/wasm/keeper/keeper_test.go [hunk 4]
```diff
@@ -1501,33 +1392,74 @@ func TestClearContractAdminInactiveContract(t *testing.T) {
...
+	handler := govKeeper.Router().GetRoute(storedProposal.ProposalRoute())
+	err = handler(ctx, storedProposal.GetContent())
+	require.NoError(t, err)
+
+	_, err = keeper.Migrate(ctx, contractAddr, creator, newCodeID, migMsgBz)
 	require.True(t, types.ErrInvalid.Is(err), "expected %v but got %+v", types.ErrInvalid, err)
 }
 
-func TestUpdateContractStatus(t *testing.T) {
+func TestUpdateContractAdminInactiveContract(t *testing.T) {
```
Comment (loloicci) [inline x/wasm/keeper/keeper_test.go:1460]: ditto.

File: x/wasm/keeper/keeper_test.go [hunk 5]
```diff
@@ -1548,87 +1480,67 @@ func TestUpdateContractStatus(t *testing.T) {
...
-		},
+	handler := govKeeper.Router().GetRoute(storedProposal.ProposalRoute())
+	err = handler(ctx, storedProposal.GetContent())
+	require.NoError(t, err)
+
+	err = keeper.UpdateContractAdmin(ctx, addr, fred, anyAddr)
+	require.True(t, types.ErrInvalid.Is(err), "expected %v but got %+v", types.ErrInvalid, err)
+}
+
+func TestClearContractAdminInactiveContract(t *testing.T) {
```
Comment (loloicci) [inline x/wasm/keeper/keeper_test.go:1503]: ditto.

File: x/wasm/keeper/keeper_test.go (hunk 2 above)
Comment (brew0722) [reply x/wasm/keeper/keeper_test.go]: right, I've edited it to have a more precise meaning. If there is a more correct sentence, please feel free to suggest it.
thks!

File: x/wasm/keeper/keeper_test.go (hunk 3 above)
Comment (brew0722) [reply x/wasm/keeper/keeper_test.go]: These tests below test whether other features are normally blocked after disabling them through the inactive gov proposal.
 I've edited it to have a more precise meaning. 
thks!

File: x/wasm/keeper/keeper_test.go (hunk 4 above)
Comment (brew0722) [reply x/wasm/keeper/keeper_test.go]: same above

File: x/wasm/keeper/keeper_test.go (hunk 5 above)
Comment (brew0722) [reply x/wasm/keeper/keeper_test.go]: same above

**Review by loloicci — APPROVED** [review]

**Review by shiki-tak — APPROVED** [review]
LGTM

**Pull request merged**